ui/src/views/Settings.vue

```diff
@@ -102,28 +153,44 @@ export default {
       q: {
         page: "settings",
       },
+      status: {},
+      validationErrorDetails: [],
       urlCheckInterval: null,
       host: "",
       isLetsEncryptEnabled: false,
+      letsEncryptIsEnabled: false,
```

Comment:
`letsEncryptIsEnabled` is too similar to `isLetsEncryptEnabled`, I suggest to rename this variable to something like `isLetsEncryptCurrentlyEnabled`